include/form_functions.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+function ValidateFormField(){
```

Comment:
similar to a comment above, this function name seems generic, but you're hardcoding in `NameBlank` and `PhoneBlank`. How could it be made more generic?

Reply:
Having said that, I see this looks like practice code, so I'm skipping the rest of this file unless you want me to review it.